ui/conductor/src/traits/lighting/LightCard.vue

```diff
@@ -115,12 +120,37 @@ const brightnessControl = computed(() => {
 });
 </script>
 
-<style scoped>
+<style lang="scss" scoped>
 .v-list-item {
   min-height: auto;
 }
 
 .v-progress-linear {
   width: auto;
 }
+
+.preset-button {
+  position: relative;
+  display: inline-flex;
+  justify-content: center;
+  width: 100%;
+  max-width: 575px; /* Adjust based on your requirement */
+  overflow: hidden;
+
+  &__container {
+    position: absolute;
+    left: 50%;
+    transform: translateX(-50%);
+    display: flex;
+    justify-content: center;
+    width: 100%;
+    max-width: 575px;
+
+    &--content {
+      white-space: nowrap;
+      overflow: hidden;
+      text-overflow: ellipsis;
+    }
+  }
+}
```

Comment:
This is _waaayyy_ more complicated than it needs to be

```html
<v-btn block ...>
  <span class="text-truncate">{{ preset.title ? preset.title : preset.name }}</span>
</v-btn>
```

Should be enough right?